client/TTTClient/TTTClient/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Remove packages from git control. They should download while compile or build process.

Reply:
Done. Now i understand purpose of this config file. Thanks.